exercises/connect/connect.py

```diff
@@ -0,0 +1,4 @@
+
+class ConnectGame:
+    def __init__(self, board):
+        pass
```

Comment:
The solution template doesn't match its usage in `connect_test.py`. The solution template should only contain placeholders for all classes and methods referenced in the tests. `connect_test.py` does not reference `connect.ConnectGame`, but it does reference `connect.play()`, so that should be in the template.

Reply:
Thanks for pointing this out! (-:
Please check the updated one.